docs/loading/Loading_data_template.md

```diff
@@ -0,0 +1,348 @@
+
+## Using Broker Load
+
+An asynchronous Broker Load process handles making the connection to S3, pulling the data, and storing the data in StarRocks.
+
+### Advantages of Broker Load
+
+- Broker Load supports data transformation, UPSERT, and DELETE operations during loading.
+- Broker Load runs in the background and clients don't need to stay connected for the job to continue.
+- Broker Load is preferred for long running jobs, the default timeout is 4 hours.
+- In addition to Parquet and ORC file format, Broker Load supports CSV files.
+
+### Data flow
+
+***Note: Processes that involve multiple components or steps may be easier to understand with a diagram. This example includes a diagram that helps describe the steps that happen when a user chooses the Broker Load option.***
+
+![Workflow of Broker Load](../assets/broker_load_how-to-work_en.png)
+
+1. The user creates a load job
+2. The frontend (FE) creates a query plan and distributes the plan to the backend nodes (BE)
+3. The backend (BE) nodes pull the data from the source and load the data into StarRocks
+
+### Typical example
+
+Create a table, start a load process that pulls a Parquet file from S3, and verify the progress and success of the data loading.
+
+> Note
+>
+> The examples use a sample dataset in Parquet format, if you want to load a CSV or ORC file, that information is linked at the bottom of this page.
+
+#### Create a table
+
+Create a database for your table:
+
+```SQL
+CREATE DATABASE IF NOT EXISTS project;
+USE project;
+```
+
+Create a table. This schema matches a sample dataset that you can load from a StarRocks S3 bucket.
+
+```SQL
+DROP TABLE IF EXISTS user_behavior;
+
+CREATE TABLE `user_behavior` (
+    `UserID` int(11),
+    `ItemID` int(11),
+    `CategoryID` int(11),
+    `BehaviorType` varchar(65533),
+    `Timestamp` datetime
+) ENGINE=OLAP 
+DUPLICATE KEY(`UserID`)
+DISTRIBUTED BY HASH(`UserID`)
+PROPERTIES (
+    "replication_num" = "1"
+);
+```
+
+#### Gather connection details
+
+> Note
+>
+> The examples use IAM-based authentication. Other authentication methods are available and linked at the bottom of this page.
+
+Loading data from S3 requires having the:
+
+- S3 bucket
+- S3 object keys (object names) if accessing a specific object in the bucket
+- S3 region
+- Access key and secret
+
+#### Start a Broker Load
+
+This `LOAD` job has four main sections:
+
+- `LABEL`: A string used when querying the state of a `LOAD` job.
+- `LOAD` declaration: The source URI, destination table, and the source data format.
+- `BROKER`: The connection details for the source.
+- `PROPERTIES`: Timeout value and any other properties to apply to this job.
+
+> Tip
```

Comment:
We have defined only NOTE, NOTICE, CAUTION, and WARNING. "Tip" is not included. So here it's better to change "Tip" to "NOTE"?

Reply:
ok.

what is the difference between NOTE and NOTICE?

Reply:
Is there a reason to use `NOTE` instead of `Note`?

Reply:
https://grammarhow.com/please-note-vs-please-notice/